client/signup/config/step-components.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
 	'get-dot-blog-themes': ThemeSelectionComponent,
 	plans: PlansStepComponent,
 	site: SiteComponent,
+	'site-or-domain': require( 'signup/steps/site-or-domain' ),
```

Comment:
@scruffian If I recall correctly, this was a workaround to prevent us from importing React components in our test suite, as the step/flow configs are used in tests.